src/MongoDB/Query.c

```diff
@@ -183,49 +363,44 @@ HashTable *php_phongo_query_get_debug_info(zval *object, int *is_temp TSRMLS_DC)
 	*is_temp = 1;
 	intern = Z_QUERY_OBJ_P(object);
 
-	array_init_size(&retval, 6);
+	array_init_size(&retval, 3);
 
-	/* Avoid using PHONGO_TYPEMAP_NATIVE_ARRAY for decoding query and selector
+	/* Avoid using PHONGO_TYPEMAP_NATIVE_ARRAY for decoding filter and opts
 	 * documents so that users can differentiate BSON arrays and documents. */
-	if (intern->query) {
+	if (intern->filter) {
 #if PHP_VERSION_ID >= 70000
 		zval zv;
 #else
 		zval *zv;
 #endif
 
-		phongo_bson_to_zval(bson_get_data(intern->query), intern->query->len, &zv);
+		phongo_bson_to_zval(bson_get_data(intern->filter), intern->filter->len, &zv);
 #if PHP_VERSION_ID >= 70000
-		ADD_ASSOC_ZVAL_EX(&retval, "query", &zv);
+		ADD_ASSOC_ZVAL_EX(&retval, "filter", &zv);
 #else
-		ADD_ASSOC_ZVAL_EX(&retval, "query", zv);
+		ADD_ASSOC_ZVAL_EX(&retval, "filter", zv);
 #endif
 	} else {
-		ADD_ASSOC_NULL_EX(&retval, "query");
+		ADD_ASSOC_NULL_EX(&retval, "filter");
 	}
 
-	if (intern->selector) {
+	if (intern->opts) {
 #if PHP_VERSION_ID >= 70000
 		zval zv;
 #else
 		zval *zv;
 #endif
 
-		phongo_bson_to_zval(bson_get_data(intern->selector), intern->selector->len, &zv);
+		phongo_bson_to_zval(bson_get_data(intern->opts), intern->opts->len, &zv);
 #if PHP_VERSION_ID >= 70000
-		ADD_ASSOC_ZVAL_EX(&retval, "selector", &zv);
+		ADD_ASSOC_ZVAL_EX(&retval, "opts", &zv);
```

Comment:
Did you use "opts" because libmongoc uses that? If not, I believe we should use "options".